ui/app/components/clients/page/counts.hbs

```diff
@@ -0,0 +1,137 @@
+        {{on "click" (fn (mut this.showBillingStartModal) true)}}
+      />
+    {{else}}
+      <DateDropdown @handleSubmit={{this.onDateChange}} @dateType="startDate" @submitText="Save" />
+    {{/if}}
+  </div>
+
+  <p class="is-8 has-text-grey has-bottom-margin-l">
+    {{this.versionText.description}}
+  </p>
+
+  {{#if (eq @activity.id "no-data")}}
+    <Clients::NoData @config={{@config}} @dateRangeMessage={{this.dateRangeMessage}} />
+  {{else if @activityError}}
+    <Clients::Error @error={{@activityError}} />
+  {{else}}
+    {{#if (eq @config.enabled "Off")}}
+      <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+        <A.Title>Tracking is disabled</A.Title>
+        <A.Description>
+          Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need
+          to
+          <Hds::Link::Inline @route="vault.cluster.clients.edit">edit the configuration</Hds::Link::Inline>
+          to enable tracking again.
+        </A.Description>
+      </Hds::Alert>
+    {{/if}}
+
+    <div class="is-subtitle-gray has-bottom-margin-m">
+      FILTERS
+      <Toolbar data-test-clients-filter-bar>
+        <ToolbarFilters>
+          <CalendarWidget
+            @startTimestamp={{this.startDate}}
+            @endTimestamp={{this.endDate}}
+            @selectMonth={{this.onDateChange}}
+          />
+          {{#if (or @namespace this.namespaces)}}
+            <SearchSelect
+              @id="namespace-search-select"
+              @options={{this.namespaces}}
+              @inputValue={{if @namespace (array @namespace)}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{fn this.setFilterValue "ns"}}
+              @placeholder="Filter by namespace"
+              @displayInherit={{true}}
+              class="is-marginless"
+            />
+          {{/if}}
+          {{#if (or @authMount this.authMounts)}}
+            <SearchSelect
+              @id="auth-method-search-select"
+              @options={{this.authMounts}}
+              @inputValue={{if @authMount (array @authMount)}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{fn this.setFilterValue "authMount"}}
+              @placeholder="Filter by auth method"
+              @displayInherit={{true}}
+            />
+          {{/if}}
+        </ToolbarFilters>
+      </Toolbar>
+    </div>
+
+    {{#if this.filteredActivity}}
+      {{#if this.startTimeDiscrepancy}}
+        <Hds::Alert data-test-clients-upgrade-warning @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+          <A.Title>Warning</A.Title>
+          <A.Description>
+            {{this.startTimeDiscrepancy}}
+          </A.Description>
+        </Hds::Alert>
+      {{/if}}
+
+      {{yield}}
```

Comment:
A block is yielded if there are counts to show. In the consuming route template there is an outlet placed in the block so that the child route can render the specific count information in the charts.